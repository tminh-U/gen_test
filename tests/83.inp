11
10 12 15 2 11 15 11 16 17 15 11 
7
4 7
1 11
3 6
7 8
4 8
11 11
11 11
