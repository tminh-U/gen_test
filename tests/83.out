12
162
10
4
24
2
2
